ProjectEuler | Update Problem statement of 002 to match projecteuler.net (#38101)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/08-coding-interview-prep/project-euler/problem-2-even-fibonacci-numbers.english.md

<section id='description'>
Each new term in the Fibonacci sequence is generated by adding the previous two terms. By starting with 1 and 2, the first 10 terms will be:
<div style='text-align: center;'>1, 2, 3, 5, 8, 13, 21, 34, 55, 89, ...</div>

> By considering the terms in the Fibonacci sequence that do not exceed number <code>n</code>, find the sum of the even-valued terms.

Reviewer: ```suggestion
By considering the terms in the Fibonacci sequence whose values do not exceed <code>n</code>, find the sum of the even-valued terms.
```
